app/src/main/java/org/oppia/android/app/databinding/DrawableBindingAdapters.java

```diff
@@ -37,28 +37,4 @@ public static void setTopBackgroundDrawable(@NonNull View view, @ColorInt int co
     GradientDrawable background = (GradientDrawable) view.getBackground();
     background.setColor(0xff000000 | colorRgb);
   }
-
-  /** Used to set a bottom rounded-rect background drawable with a data-bound color. */
-  @BindingAdapter("app:bottomRoundedRectDrawableWithColor")
-  public static void setBottomBackgroundDrawable(@NonNull View view, @ColorInt int colorRgb) {
```

Comment:
Why are these being removed?

Reply:
@BenHenning @rt4914 https://github.com/oppia/oppia-android/pull/4077#issuecomment-1004925233
I have removed these function because these functions are not being used anywhere. So as mentioned in this [comment](https://github.com/oppia/oppia-android/pull/4077#issuecomment-1004925233) I have removed them

Reply:
![image](https://user-images.githubusercontent.com/78549195/150001947-21276fbf-7d13-45aa-b395-fe018637e6aa.png)

Reply:
I thought I remembered you mentioning them. Did you double check that the adapter binding name isn't being used, too? I.e. the ``app:bottomRoundedRectDrawableWithColor``?

Reply:
![image](https://user-images.githubusercontent.com/78549195/150642038-a7acdd23-f9a2-4df6-b272-2bb538e699fd.png)

Reply:
Got it--thanks for double checking.